components/script/security_manager.rs

```diff
@@ -224,10 +225,7 @@ impl FetchResponseListener for CSPReportUriFetchListener {
         submit_timing(self, CanGc::note())
     }
 
-    fn process_csp_violations(&mut self, _request_id: RequestId, violations: Vec<Violation>) {
-        let global = &self.resource_timing_global();
```

Comment:
I realized that trying to send CSP errors about an endpoint to send CSP errors to would result in an infinite loop. Given that we also don't send any for `report-to`, this seems like a footgun.